acceptance/tests/language/sensitive_data_type.rb

```diff
@@ -75,13 +82,20 @@
           run_assertions(assertion_code, result)
         end
 
-        if agent['platform'] =~ /fedora|centos|el|redhat|scientific|ubuntu|debian|cumulus/
-          step "assert no redacted data in syslog" do
-            #sigh.  gee, thanks for the help, beaker!
-            result = dump_puppet_log(agent)
```

Comment:
Yeah, definitely not great to be duplicating syslog paths here.  Would be much better to have that abstracted across OSes by beaker.